_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -0,0 +1,280 @@
+- [H57: Using language attributes on the html element](https://www.w3.org/WAI/WCAG21/Techniques/html/H57)
+- [BCP 47: Tags for Identifying Languages](https://www.ietf.org/rfc/bcp/bcp47.txt)
+- [The `lang` and `xml:lang` attributes](https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This page has a [default language][default page language] of English, which matches the language indicated in the `lang` attribute.
+
+```html
+<html lang="en">
+	<head>
+		<title>ACT Rules Format 1.0 - Abstract</title>
+	</head>
+	<body>
+		<p>
+			The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test
+			rules. These test rules can be used for developing automated testing tools and manual testing methodologies. It
+			provides a common format that allows any party involved in accessibility testing to document and share their
+			testing procedures in a robust and understandable manner. This enables transparency and harmonization of testing
+			methods, including methods implemented by accessibility test tools.
+		</p>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+This page has a [default language][default page language] of English because most words are English. The [default page language][] matches the language indicated in the `lang` attribute.
+
+```html
+<html lang="en">
+	<head>
+		<title>Gelukkig</title>
+	</head>
+	<body>
+		<p>The Dutch word "gelukkig" has no equivalent in English.</p>
+	</body>
+</html>
+```
+
+#### Passed Example 3
+
+This page has a [default language][default page language] of Dutch because all English text is wrapped in a `p` element with a `lang` attribute set to `en`. Dutch ("nl") is set as the language of the page by the `lang` attribute on the `html` element.
+
+```html
+<html lang="nl">
+	<head>
+		<title>Met de kippen op stok</title>
+	</head>
+	<body>
+		<blockquote>
+			<p>"Hij ging met de kippen op stok"</p>
+		</blockquote>
+		<p lang="en">
+			This Dutch phrase literally translates into "He went to roost with the chickens", but it means that he went to bed
+			early.
+		</p>
+	</body>
+</html>
+```
+
+#### Passed Example 4
+
+This page has a [default language][default page language] of English because the `title` element and the [accessible name][] of the `img` element are English. The [visible][] text Dutch text does not count as its parent element has a `lang` attribute.
+
+```html
+<html lang="en">
+	<head>
+		<title>Fireworks over Paris</title>
+	</head>
+	<body>
+		<img src="/test-assets/shared/fireworks.jpg" alt="Fireworks over Paris" />
+		<p lang="nl">
+			Gelukkig nieuwjaar!
+		</p>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This page has a [default language][default page language] of English, which is not the language indicated in the `lang` attribute.
```

Comment:
We are not explaining why this page has a default language of English. Adding "as all of its words are English" or something along these lines may help clarity and consistency.

Reply:
I don't think we need to explain that. How definitions work rarely gets explained unless it is essential for understanding the example.

Reply:
I get that it is not "essential" to understand the example, but I do think it would help. Mildly thinking these explanations should be added.